internal/handlers/utils_test.go

```diff
@@ -0,0 +1,137 @@
+package handlers
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net/http"
+	"testing"
+	"time"
+
+	"github.com/docker/docker/api/types/image"
+	"github.com/stretchr/testify/require"
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/modules/registry"
+	"github.com/testcontainers/testcontainers-go/wait"
+)
+
+const (
+	registryPort = "5000/tcp"
+	authUser     = "user"
+	authPass     = "password"
+	registryURI  = "ghcr.io/rancher-sandbox/sbombastic/test-assets"
+	imageName    = "golang"
+	tag          = "1.12-alpine"
+	platform     = "linux/amd64"
+	digest       = "sha256:1782cafde43390b032f960c0fad3def745fac18994ced169003cb56e9a93c028"
+)
+
+type RegistryTestSuite struct {
```

Comment:
The current name is ambiguous. Considering the method names, it could easily be confused with the `suite` from [testify](https://pkg.go.dev/github.com/stretchr/testify/suite). Would it be clearer to rename this  `testPrivateRegistry` (no need to be public)?